easybuild/easyconfigs/s/statsmodels/statsmodels-0.9.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonBundle'
+
+name = 'statsmodels'
+version = '0.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.statsmodels.org'
+description = """Statsmodels is a Python module that provides classes and functions
+for the estimation of many different statistical models, as well as for conducting 
```

Comment:
I suppose it's not mandatory (@boegel?) but indenting (with 1 space) the continuation lines in multi-line descriptions makes the easyconfig easier to read... (also, as it is there is no space between "functions" and "for", only a newline character)

Reply:
fixed.